rsopt/pkcli/pack.py

```diff
@@ -86,21 +96,27 @@ def _create_tar(name, file_list):
 
     return tarname
 
+
 def configuration(config, ignore=None, add=None):
-    file_list = [config,]
+    file_list = [config, ]
+    if not ignore:
+        ignore = []
     config_yaml = parse.read_configuration_file(config)
     _config = parse.parse_yaml_configuration(config_yaml)
 
     for job in _config.jobs:
         files = _get_files_from_job(job)
+        ignore.extend(job.ignored_files(with_path=True))
         file_list.extend(files)
 
     files = _get_files_from_options(_config)
     file_list.extend(files)
+    # remove any duplicate items
+    file_list = set(file_list)
 
     if ignore:
         for file in ignore:
-            if ignore in file_list:
+            if file in file_list:
                 file_list.remove(file)
             else:
                 print(f'File: {file} is not being used by rsopt configuration')
```

Comment:
`f'"Ignored file: {file}...` instead for a bit more useful message

Should this actually raise an exception? Or is it okay having an ignored file that isn't there for a production run?

Reply:
If you request a file to be ignored and it isn't there then nothing is really broken. The message is more of a courtesy. I don't think an error should be raised in this case.